cmake/modules/RootBuildOptions.cmake

```diff
@@ -143,11 +143,7 @@ ROOT_BUILD_OPTION(opengl ON "Enable support for OpenGL (requires libGL and libGL
 ROOT_BUILD_OPTION(oracle ON "Enable support for Oracle databases (requires Oracle Instant Client)")
 ROOT_BUILD_OPTION(pgsql ON "Enable support for PostgreSQL")
 ROOT_BUILD_OPTION(pyroot ON "Enable support for automatic Python bindings (PyROOT)")
-if(WIN32)
-  ROOT_BUILD_OPTION(pyroot_experimental OFF "Use experimental Python bindings for ROOT")
-else()
-  ROOT_BUILD_OPTION(pyroot_experimental ON "Use experimental Python bindings for ROOT")
-endif()
+ROOT_BUILD_OPTION(pyroot_experimental ON "Use experimental Python bindings for ROOT")
```

Comment:
@bellenot is this what we want?

Reply:
No, see the full commit. The way to adjust platform defaults is to turn it off below.